src/api/ledger/utils.js

```diff
@@ -105,6 +105,28 @@ function hasCompleteLedgerRange(remote: Remote, minLedgerVersion?: number,
     maxLedgerVersion || remote.getLedgerSequence());
 }
 
+function hasLedger(remote: Remote, ledger: ?number) {
+  return remote.getServer().hasLedger(ledger);
+}
+
+function isNonContiguousLedgerRange(remote: Remote, minLedgerVersion: ?number,
+                                    maxLedgerVersion: ?number
+): boolean {
+  if (!minLedgerVersion || !maxLedgerVersion) {
+    return false;
```

Comment:
I was thinking delete `isNonContigousLedgerRange` because it's just the `else` case above whenever its not a pending error (doesn't need to be an else if).